compiler/include/stmt.h

```diff
@@ -62,7 +62,7 @@ class ResolveScope;
 class VisibilityStmt : public Stmt {
  public:
            VisibilityStmt(AstTag astTag);
-  virtual ~VisibilityStmt();
+  ~VisibilityStmt() override;
```

Comment:
Ditto aligning V under V.
I suggest indenting both the constructor and the destructor to the left, as they no longer need to be in the middle.